wgsl/index.bs

```diff
@@ -11071,6 +11078,9 @@ example, a memory read that accesses a [=u32=] from a struct containing
 multiple members, only reads the memory locations associated with that u32
 member.
 
+Note: A write access to a component of a vector **may** access all memory locations
```

Comment:
Right.   From a memory model perspective, it doesn't matter if extra reads are performed as part of the right. If there was a race assuming only writes, they would still be a race if you added a read at that point.